sensor/kubernetes/listener/resources/hash_reconciliation.go

```diff
@@ -0,0 +1,50 @@
+package resources
+
+import (
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/stringutils"
+)
+
+// InMemoryStoreReconciler handles sensor-side reconciliation using in-memory store
+type InMemoryStoreReconciler struct {
```

Comment:
nit: There was a recent change that made `InMemoryStores` be not-so-in-memory since they are backed by `emptyDir`. I planned on changing the name for a while but didn't get to it yet. Maybe let's avoid using it here too. 

So how about:
```suggestion
type ResourceStoreReconciler struct {
```